schema/types/suricata.schema

```diff
@@ -351,6 +373,160 @@ type suricata.krb5 = record {
   host: string
 }
 
+type suricata.topicitem = record {
+  topic: string,
+  qos: count
+}
+
+type suricata.mqtt = record {
+  timestamp: timestamp,
+  flow_id: count #index=hash,
+  pcap_cnt: count,
+  vlan: list<count>,
+  in_iface: string,
+  src_ip: addr,
+  src_port: port,
+  dest_ip: addr,
+  dest_port: port,
+  proto: string,
+  event_type: string,
+  community_id: string #index=hash,
+  mqtt: record {
+    publish: record {
+      qos: count,
+      retain: bool,
+      dup: bool,
+      truncated: bool,
+      skipped_length: count,
+      topic: string,
+      message_id: count,
+      message: string,
+//      properties:map<string, string>
+    },
+    pubrec: record {
+      qos: count,
+      retain: bool,
+      dup: bool,
+      truncated: bool,
+      skipped_length: count,
+      message_id: count,
+      reason_code: count,
+//      properties:map<string, string>
+    },
+    pubrel: record {
+      qos: count,
+      retain: bool,
+      dup: bool,
+      truncated: bool,
+      skipped_length: count,
+      message_id: count,
+      reason_code: count,
+//      properties:map<string, string>
+    },
+    pubcomp: record {
+      qos: count,
+      retain: bool,
+      dup: bool,
+      truncated: bool,
+      skipped_length: count,
+      message_id: count,
+      reason_code: count,
+//      properties:map<string, string>
+    },
+    connect: record {
+      qos: count,
+      retain: bool,
+      dup: bool,
+      truncated: bool,
+      skipped_length: count,
+      protocol_string: string,
+      protocol_version: count,
+      flags: record {
+        username: bool,
+        password: bool,
+        will_retain: bool,
+        will: bool,
+        clean_session: bool
+      },
+      client_id: string,
+      username: string,
+      password: string,
+      will: record {
+        topic: string,
+        message: string,
+//        properties:map<string, string>
+      },
+//      properties:map<string, string>
+    },
+    connack: record {
+      qos: count,
+      retain: bool,
+      dup: bool,
+      truncated: bool,
+      skipped_length: count,
+      session_present: bool,
+      return_code: count,
+//      properties:map<string, string>
+    },
+    subscribe: record {
+      qos: count,
+      retain: bool,
+      dup: bool,
+      truncated: bool,
+      skipped_length: count,
+      message_id: count,
+//      topics: list<suricata.topicitem>,
+//      properties:map<string, string>
```

Comment:
```suggestion
//    properties: map<string, string>,
```

Style nits: Remove two spaces, space after colon, and trailing comma.

Reply:
Done.